src/components/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import Flex from "@reearth/components/atoms/Flex";
+import Text from "@reearth/components/atoms/Text";
+import { styled, useTheme } from "@reearth/theme";
+
+export type Size = "small" | "large";
+
+export type Props = {
+  className?: string;
+  size?: Size;
+  color?: string;
+  avatar?: string;
+  innerText?: string | number;
```

Comment:
The inner text should be string always, make the transform to string before passing it.
It's not meant to have `innerText` with a `number` type